src/vasc/properties.cpp

```diff
@@ -65,8 +65,8 @@ static bool compare_section_structure(const std::vector<VascSection::Type>& vec1
     if (vec1.size() != vec2.size()) {
         if (verbose_)
             printError(Warning::UNDEFINED,
-                    "Error comparing " + name + ", size differs: " + std::to_string(vec1.size()) +
-                        " vs " + std::to_string(vec2.size()));
+                       "Error comparing " + name + ", size differs: " +
+                           std::to_string(vec1.size()) + " vs " + std::to_string(vec2.size()));
         return false;
```

Comment:
Curious about such formatting changes : is this result from different settings for formatting? or different clang-format?

Reply:
I'm using the .clang_format file included in the project and I used CLION to reapply it to the whole project. Does it look off to you?

Reply:
>  or different clang-format?

If they differ, that's the problem.  The included `ci/check_clang_format.sh` pins the clang-format at `9.0.0` at the moment, so differences show up if you're not using the same version.

Reply:
Thanks for the clarification. I will investigate this.

Reply:
I've checked and clang format 9.0.0 seems to be there. Are there guidelines on how you apply .clang-format in this project? or better, a pre-made script? Thanks!

Reply:
Running `./ci/check_clang_format.sh` should be everything you need.